Heroku/README.md

```diff
@@ -49,9 +49,17 @@ A cheat sheet for deployment:
 
 ```console
```

Comment:
This still doesn't show output, so `sh` and no dollars would be better. The comments can stay.

But this can wait!